src/macosx/system.m

```diff
@@ -66,7 +66,7 @@ static void osx_tell_dock(void)
 {
    ProcessSerialNumber psn = { 0, kCurrentProcess };
    TransformProcessType(&psn, kProcessTransformToForegroundApplication);
-   [NSApplication sharedApplication];
+   [[NSApplication sharedApplication] activateIgnoringOtherApps: YES];
```

Comment:
This is neither here nor there, but I just have to say that Objective-C is a very strange language. :-p